subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -1155,17 +1162,39 @@ static uint8_t ptc_calc(const struct lll_adv_iso *lll, uint32_t event_spacing,
 			uint32_t event_spacing_max)
 {
 	if (event_spacing < event_spacing_max) {
+		uint8_t nse;
 		uint8_t ptc;
 
 		/* Possible maximum Pre-transmission Subevents per BIS */
 		ptc = ((event_spacing_max - event_spacing) /
 		       (lll->sub_interval * lll->bn * lll->num_bis)) *
 		      lll->bn;
 
-		/* FIXME: Here we restrict to a maximum of BN Pre-Transmission
-		 * subevents per BIS
+		/* Required NSE */
+		nse = lll->bn * lll->irc;
```

Comment:
Ditto here; risk of overflow when multiplying 2 uint8_ts?

Reply:
No, `bn` is 3 bits and `irc` is 4 bits.